app-office/texmacs/texmacs-1.99.12.recipe

```diff
@@ -0,0 +1,111 @@
+SUMMARY="A scientific editing platform designed to create beautiful technical documents"
+DESCRIPTION="TeXmacs provides a unified and user friendly framework for editing \
+structured documents with different types of content: text, mathematics, graphics, \
+interactive content, slides, etc.\
+Its rendering engine uses high-quality typesetting algorithms so as to produce \
+professionally looking documents, which can either be printed out or presented from \
+a laptop."
+HOMEPAGE="http://www.texmacs.org"
+COPYRIGHT="Free Software Foundation"
+LICENSE="GNU GPL v3"
+
+SOURCE_URI="https://github.com/mgubi/texmacs/archive/texmacs-$portVersion-haiku-qt5.tar.gz"
+CHECKSUM_SHA256="607fa4db4676b78780f7af086feec2e2332dc18a371d79fa50d640de6db10b43"
+SOURCE_DIR="texmacs-texmacs-$portVersion-haiku-qt5/src"
+PATCHES="texmacs-$portVersion.patchset"
+
+REVISION="1"
+
+ADDITIONAL_FILES="texmacs.rdef.in"
+DISABLE_SOURCE_PACKAGE="yes"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	texmacs$secondaryArchSuffix = $portVersion
+	app:TeXmacs
+	cmd:fig2ps$secondaryArchSuffix
+	"
+
+REQUIRES="
```

Comment:
Why libguile not listed here and in BUILD_REQs?

Reply:
I was not sure what was the proper way. I will add guile1 (which is my package for guile 1.8, I submitted another PR for that). TeXmacs is not yet compatible with Guile 2.2.